presto-tests/src/test/java/io/prestosql/tests/AbstractTestEngineOnlyQueries.java

```diff
@@ -3391,6 +3391,18 @@ public void testLogicalExplain()
         assertEquals(getOnlyElement(result.getOnlyColumnAsSet()), getExplainPlan(query, LOGICAL));
     }
 
+    @Test
+    public void testJoinDistribution()
+    {
+        String query = "SELECT c.custkey\n" +
+                       "FROM customer c JOIN nation n\n" +
+                       "  ON n.nationkey = c.nationkey\n" +
+                       "WHERE n.name = 'UNITED STATES'\n" +
+                       "LIMIT 5";
+        MaterializedResult result = computeActual("EXPLAIN (FORMAT TEXT) " + query);
+        assertThat(result.getMaterializedRows().get(0).toString()).matches("(?s).*Distribution:.*");
```

Comment:
Since the result should only have one row containing one column, you can use `result.getOnlyValue()`.